backbone.js

```diff
@@ -1128,7 +1128,7 @@
   // right here:
   var collectionMethods = { forEach: 3, each: 3, map: 3, collect: 3, reduce: 4,
       foldl: 4, inject: 4, reduceRight: 4, foldr: 4, find: 3, detect: 3, filter: 3,
-      select: 3, reject: 3, every: 3, all: 3, some: 3, any: 3, include: 2,
+      select: 3, reject: 3, every: 3, all: 3, some: 3, any: 3, include: 2, includes: 3,
```

Comment:
I only did it for `includes` since it's the meant to emulate the ES6 method, but I could see putting the others to `3` as well.